config/locales/en.yml

```diff
@@ -20,6 +20,7 @@ en:
         edit: 'Update property'
         index: 'Properties'
         show: '%{name}'
+        find: 'Find a property'
```

Comment:
I know we briefly chatted about this and as I've looked at it I would say that it might make more sense for this copy to live on the page it appears, which I think in this case would be the dashboard?